R/deprecated.R

```diff
@@ -1,24 +1,20 @@
+## ourPkg-deprecated.r
+#' @title Deprecated functions in package \pkg{DeclareDesign}.
+#' @description The functions listed below are deprecated and will be defunct in
+#'   the near future. When possible, alternative functions with similar
+#'   functionality are also mentioned. Help pages for deprecated functions are
+#'   available at \code{help("-deprecated")}.
+#' @name ourPkg-deprecated
+#' @keywords internal
+NULL
 
-#' Deprecated functions
+#' @rdname ourPkg-deprecated
+#' @section \code{get_estimands}:
+#' For \code{get_estimands}, use \code{\link{draw_estimands}}.
 #'
-#' The function \code{get_estimands} has been replaced with \code{draw_estimands}.
-#'
-#' @param ... options sent to the old version of \code{get_estimands}.
+#' @importFrom lifecycle deprecate_stop
 #'
 #' @export
-#'
-#' @name deprecated
-get_estimands <- function(...) {
-  qs <- quos(...)
-  
-  .Deprecated(
-    new = "draw_estimands",
-    old = "get_estimands",
-    msg = paste0(
-      "\nThe get_estimands function has been renamed draw_estimands, to make clear ",
-      "that the estimands are draws from a simulation. Please replace your code with draw_estimands."
-    )
-  )
-  
-  cat()
+get_estimands <- function(design) {
+  deprecate_stop("1.0.0", "get_estimands()", "draw_estimands()")
```

Comment:
This has been marked deprecated for a while, you can safely remove it at this point. I would add an alias to draw_estimands roxygen so that ?get_estimands resolves to the new function.